2-alcance/README.md

```diff
@@ -1,3 +1,286 @@
 # Plano de alcance
 
-## Referencias
\ No newline at end of file
+## Introducción (Angie)
+Todo proyecto web requiere de objetivos, estrategias y metas claras para que sea un éxito. Debido a ello y por medio de este trabajo se desarrollarán las especificaciones funcionales de manera detallada y así ahorrar tiempo en la elaboración del diseño del nuestro proyecto denominado Aupa. 
+
+## Especificaciones funcionales 
```

Comment:
En este apartado de especificaciones funcionales, en realidad no habláis de otra cosa que no sea contenido. Lo único que describís es la manera en la que puede navegarse de un contenido a otro, con una descripción del contenido de cada sección (que, por cierto, no corresponde en este apartado).

Revisad el contenido sobre especificaciones funcionales. Lo que debéis definir aquí es qué tareas podrá el usuario llevar a cabo en vuestro sitio web: puede navegar por el contenido, pero ¿algo más? ¿Registrarse? ¿Suscribirse a nuevas publicaciones? ¿Utilizar un buscador avanzado? ¿Enviar algún mensaje mediante un formulario? ¿...?